src/middlewared/middlewared/plugins/system.py

```diff
@@ -98,6 +101,21 @@ class Config:
                         'Serial port specified has not been identified by the system'
```

Comment:
Not clear what "identified" means.  Probably "is not present on this system."

Reply:
Not part of the PR so we can handle it in another PR/ticket.